worker/include/Utils.hpp

```diff
@@ -86,6 +86,8 @@ namespace Utils
 		}
 
 		static void NormalizeIp(std::string& ip);
+
+		static bool IsMulticast(const struct sockaddr_storage* addr, const int& family);
```

Comment:
No need to pass `family` by reference. The method should NOT be able to change its original value.
```suggestion
		static bool IsMulticast(const struct sockaddr_storage* addr, const int family);
```